### PR TITLE
Fixing comments length.

### PR DESCRIPTION
**Pull request by aleksanderziel**

### CODE REVIEW

**Review by zie1ony — CHANGES_REQUESTED** [review]
Good job, but fix all the lines.

File: README.md [hunk 1]
```diff
@@ -22,13 +24,15 @@ To run test on your local machine, you can basically execute the command:
...
 ```
 $ cargo odra test -b casper
 ```
 
 ### Backends
 
-To build or test against a specific backend, you need pass a -b parameter to `cargo-odra`.
+To build or test against a specific backend, 
+you need pass a -b parameter to `cargo-odra`.
 A list of available backends can check [here](https://github.com/odradev/odra).
```
Comment (zie1ony) [inline README.md:38]:
```suggestion
A list of available backends can check [here](https://github.com/odradev/odra#backends).
```
This links directly to the backend section.

File: src/flipper.rs [hunk 2]
```diff
@@ -42,8 +46,10 @@ mod tests {
 
     #[test]
     fn flipping() {
-        // To test a module we need to deploy it. To do so, Odra generate for us deploy() function.
-        // To call a constructor we don't do it directly. In this case to call a constructor, we would call deploy_initial_settings() function.
+        // To test a module we need to deploy it. 
+        /// To do so, Odra generate for us deploy() function.
+        // To call a constructor we don't do it directly. In this case to call..
+        //... a constructor, we would call deploy_initial_settings() function.
```
Comment (zie1ony) [inline src/flipper.rs:52]: What is the case with thos dots?

**Review by zie1ony — CHANGES_REQUESTED** [review]
Changes needed.

File: src/flipper.rs (hunk 2 above)
Comment (aleksanderziel) [reply src/flipper.rs:52]: Corrected.

File: src/flipper.rs [hunk 3]
```diff
@@ -42,8 +46,10 @@ mod tests {
 
     #[test]
     fn flipping() {
-        // To test a module we need to deploy it. To do so, Odra generate for us deploy() function.
-        // To call a constructor we don't do it directly. In this case to call a constructor, we would call deploy_initial_settings() function.
+        // To test a module we need to deploy it. 
+        /// To do so, Odra generate for us deploy() function.
```
[the remark this answers is not in the document]
Comment (aleksanderziel) [reply src/flipper.rs:50]: Corrected.

File: README.md (hunk 1 above)
Comment (aleksanderziel) [reply README.md:38]: Done.

**Review by zie1ony — APPROVED** [review]
+1

**Pull request merged**